app/models/match_prioritization/days_homeless_veterans_families_youth.rb

```diff
@@ -20,8 +20,20 @@ def self.prioritization_for_clients(scope, match_route:) # rubocop:disable Lint/
       scope.order(vets, family, is_youth, age_youth, days)
     end
 
+    def self.client_prioritization_value_label
+      'Cumulative days homeless'
+    end
+
     def self.client_prioritization_value_method
-      'days_homeless_veterans_families_youth'
+      'days_homeless'
```

Comment:
@awisema I updated these, which I think looks nicer. This is going to affect the "score" that's populated in this match census thing. Previously it would have been the long description string, now it is the days homeless value. Is that okay? https://github.com/greenriver/boston-cas/blob/3f61ca23ea227cb86cbf8835fd6f3a1868608233/app/models/match_census.rb#L28-L35


<img width="1276" alt="Screen Shot 2022-08-25 at 11 30 08 AM" src="https://user-images.githubusercontent.com/5333982/186708061-4a7cf9f6-2cec-4da5-a595-345caf2916f2.png">

Reply:
https://github.com/greenriver/boston-cas/pull/643/commits/27d6f629fc2ed0b8383609e742d1d1666f255264 updated as discussed @eanders @awisema, so a summary string is included in the MatchCensus.

```
=> #<MatchCensus:0x0000ffff8b882c10
 id: 1,
 date: Mon, 29 Aug 2022,
 opportunity_id: 17,
 match_id: 36,
 program_name: "Housing the Homeless",
 sub_program_name: "Sponsored",
 prioritized_client_ids:
  [{"score"=>"Adult and Child: true; Veteran: false; Youth: false; Cumulative days homeless: 77", "client_id"=>6043},
   {"score"=>"Adult and Child: false; Veteran: true; Youth: true; Cumulative days homeless: 3627", "client_id"=>3470},
   {"score"=>"Adult and Child: false; 
```

```
=> #<MatchCensus:0x0000ffff7f1991a8
 id: 2,
 date: Mon, 29 Aug 2022,
 opportunity_id: 17,
 match_id: 36,
 program_name: "Housing the Homeless",
 sub_program_name: "Sponsored",
 prioritized_client_ids:
  [{"score"=>1080, "client_id"=>3538},
   {"score"=>1080, "client_id"=>3076},
   {"score"=>1080, "client_id"=>3211},
   {"score"=>1080, "client_id"=>3544},
   {"score"=>1080, "client_id"=>3071},
```